src/terms/pairwise.jl

```diff
@@ -37,7 +43,8 @@ end
                                                    basis::PlaneWaveBasis{T}, ψ, occ;
                                                    kwargs...) where {T}
     forces = zero(basis.model.positions)
-    energy_pairwise(basis.model, term.V, term.params; term.max_radius, forces)
+    energy_pairwise(basis.model, term.V, term.params; max_radius=term.max_radius,
```

Comment:
why? We use the previous convention quite a lot, it's pretty nice in simplifying code. If your concern is that it somewhat hides the nature of the arguments, you can line-break on the `;`